### PR TITLE
Run cythonize upon build

### PR DESCRIPTION
**Pull request by akloboucnik**
This should fix the issue #52

### CODE REVIEW

**Comment by akloboucnik** [comment]
@superbobry Hi! Any chance this will get merged?

**Comment by timothyjlaurent** [comment]
Yes, this is breaking any dependent builds... Please fix

**Comment by morenoh149** [comment]
what checks?

File: setup.py [hunk 1]
```diff
@@ -45,13 +68,6 @@
...
-          Extension("datrie", [
-              'src/datrie.c',
-              'src/cdatrie.c',
-              'src/stdio_ext.c'
-          ], include_dirs=[LIBDATRIE_DIR])
-      ],
-
-      setup_requires=["pytest-runner"],
+      ext_modules=ext_modules,
+      setup_requires=["pytest-runner", 'Cython>=0.28;python_version<="3.7"'],
```
Comment (superbobry) [inline setup.py]: Is there a reason to upper-bound the version here?

**Comment by tacaswell** [comment]
This got merged in #80 .  Thank you for your work on this @akloboucnik !

**Pull request closed without merging**